charts/gardener/controlplane/charts/application/templates/rbac-user.yaml

```
@@ -444,7 +444,7 @@ subjects:
 apiVersion: rbac.authorization.k8s.io/v1
 kind: Role
 metadata:
-  name: gardener.cloud:system:read-gardener-info-configmap
+  name: gardener.cloud:system:read-gardener-system-public-configmaps
```

```suggestion
  name: gardener.cloud:system:read-gardener-system-info
```

Let's eliminate the resource kind. Otherwise we'll need to adjust the name again with the next change.